rootchecks/cis_win2012r2_memberL1_rcl.txt

```diff
@@ -250,7 +250,7 @@ r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa -> EveryoneIncludesAno
 #
 #2.3.10.6 Configure 'Network access: Named Pipes that can be accessed anonymously'
 [CIS - Microsoft Windows Server 2012 R2 - 2.3.10.6: Configure 'Network access: Named Pipes that can be accessed anonymously'] [any] [https://workbench.cisecurity.org/benchmarks/288]
-r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Services\LanManServer\Parameters -> NullSessionPipes -> !r:lsarpc|netlogon|samr;
+r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Services\LanManServer\Parameters -> NullSessionPipes -> !r:\.;
```

Comment:
Is this correct? `\.` matches only a single character.  Did you mean `\.+` ?

Reply:
I am sorry, there shouldn't be an exclamation mark. This registry value is not allowed to have any data on a member server, so r:\. (without !) should be correct in my opinion. But i have tested this today with my ossec 3.1.0 environment and it does not work. It seems to me that a none data value is identified as data with spaces. So i will suggest:

```suggestion
r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Services\LanManServer\Parameters -> NullSessionPipes -> r:\S*;
```
I am sorry for inconvenience!